tests/test_integration.py

```diff
@@ -13,18 +13,31 @@
 
 NATIVE_TOXENV = f"py{sys.version_info[0]}{sys.version_info[1]}"
 NATIVE_EXECUTABLE = str(pathlib.Path(sys.executable).resolve())
-TOX_INI = pathlib.Path(__file__).parent / "tox.ini"
-DOT_TOX = pathlib.Path(__file__).parent / ".tox"
+FIXTURES_DIR = pathlib.Path(__file__).parent / "fixtures"
+DOT_TOX = pathlib.Path("./.tox")
 
 
-def tox(*args, prune=True, **kwargs):
-    if prune:
-        shutil.rmtree(DOT_TOX, ignore_errors=True)
+@pytest.fixture(autouse=True)
+def projdir(tmp_path):
+    pwd = tmp_path / "projdir"
+    pwd.mkdir()
+    for fname in "tox.ini", "setup.py":
+        shutil.copy(FIXTURES_DIR / fname, pwd)
+
+    old_pwd = os.getcwd()
+    try:
+        os.chdir(pwd)
+        yield
+    finally:
+        os.chdir(old_pwd)
```

Comment:
pytest has [`monkeypatch.chdir()`](https://docs.pytest.org/en/latest/monkeypatch.html) for this.

Reply:
added a fixup commit